src/tests/ftest/pool/storage_ratio.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from apricot import TestWithServers
+from test_utils_pool import TestPool
+from avocado.core.exceptions import TestFail
+
+class StorageRatio(TestWithServers):
+    """Storage Ratio test cases.
+
+    Test class Description:
+        Verify the Storage ratio is getting checked during creation.
+
+    :avocado: recursive
+    """
+    def test_storage_ratio(self):
+        """Jira ID: DAOS-2332.
+
+        Test Description:
+            Purpose of this test is to verify minimum required SCM/NVME
+            storage space for pool creation.
+
+        Use case:
+        Create Pool with different SCM/NVMe pool size and verify that pool
+        creation failed if SCM storage size is too low compare to NVMe size.
+        For now 6% minimum SCM size needed against NVMe. There is no Maximum
```

Comment:
@johannlombardi requested a minimum ratio of 1% rather than 6% (that was the threshold that triggered the warning that existed before it was removed)